src/main/php/CanddiAI/Service/Lookup/Abstract.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
@mewtlu can you get rid of the Service part of this namespace / folder

namespace CanddiAI\Lookup;